consensus/blake3/consensus.go

```diff
@@ -44,6 +44,7 @@ var (
 
 	allowedFutureBlockTimeSeconds = int64(15) // Max seconds from current time allowed for blocks, before they're considered future blocks
 	maxUncles                     = 2         // Maximum number of uncles allowed in a single block
+	fakeDifficulties              = []*big.Int{new(big.Int).Mul(params.MinimumDifficulty, big.NewInt(4)), new(big.Int).Mul(params.MinimumDifficulty, big.NewInt(2)), params.MinimumDifficulty}
```

Comment:
You should not have any changes in `consensus/*`. The reason these diffs are here, is because you did not use a cherry-pick when you pulled in my Blake3 faker fixes. Cherry-pick would have enabled git to ignore these diffs, because git can see that cherry-picked commit already came in another PR.

To resolve this, you're going to need to revert these files and rebase.